### ISSUE TITLE
Synchronization is stuck

### ISSUE THREAD

**Issue by grittibaenz** [iotaledger/iri#850]
### Bug description
My IOTA IRI node stopped updating the `latestSolidSubtangleMilestone` when the new version `1.5.0` was released. Also after updating to the latest version the synchronization is still stuck.

I downloaded the latest database snapshot several times but the synchronization never continues. As recommended by neighbors I also left the node running for many hours (48h at least). The `Latest Milestone Tracker` keeps reporting that the latest milestone has changed yet the `solid milestone index` still does not change. I also started the node with the flag `--revalidate` as recommended by members in the IOTA Discord chat. When doing so it keeps updating the `solid milestone index` for a few blocks, but then again stops updating.

### Hardware Spec
The node is running inside of a Docker container (CentOS 7.5) on-top of a virtual machine (CentOS 7.5).
16 cores (Intel Xeon E5)
16 GB memory
80 GB SSD (Raid5, 500MB/s each)

### Steps To Reproduce
1. Download the latest database snapshot from `http://db.iota.partners/IOTA.partners-mainnetdb.tar.gz`
2. Unarchive the database
3. Start the node with the `--revalidate` flag (inside of a Docker container?)

### Expected behaviour
The node must update the `latestSolidSubtangleMilestone` after connecting to its peers.

### Actual behavior
The node connects to its peers, reports that the `latestSubtangleMilestone` has changed but never updates the `latestSolidSubtangleMilestone`.

### Errors
The node does not report any error messages (except that 1 neighbors is down) but suddenly stops updating the `latestSolidSubtangleMilestone`. Please find the log output below:
https://paste.ee/p/hZ5Ac

### Other
Please find the configuration file of my node below:
```
[IRI]
PORT = 14265
UDP_RECEIVER_PORT = 14600
TCP_RECEIVER_PORT = 15600
API_HOST = 0.0.0.0
IXI_DIR = /app/iri/ixi
HEADLESS = true
DEBUG = false
TESTNET = false
DB_PATH = /data/iri
RESCAN_DB = false
ZMQ_ENABLED = true
ZMQ_PORT = 5556

REMOTE_LIMIT_API = "removeNeighbors, addNeighbors, getNeighbors, setApiRateLimit"

NEIGHBORS = tcp://host01.domain.tld:15600 tcp://host02.domain.tld:15600 tcp://host03.domain.tld:15600  udp://host04.domain.tld:14400 udp://host05.domain.tld:14600
```

Please find the Dockerfile below:
```
FROM centos:centos7

EXPOSE 14265/tcp
EXPOSE 14600/udp
EXPOSE 15600/tcp

RUN yum -y install java-1.8.0 && \
    yum -y install wget && \
    yum clean all && \
    rm -rf /var/cache/yum

VOLUME /data/iri/

RUN install --directory --owner=root --group=root --mode=0755 /app/iri/bin && \
    install --directory --owner=10000 --group=10000 --mode=0755 /app/iri/ixi

RUN useradd -c 'iri' -m -p '!' -u 10000 -s '/bin/bash' iri

RUN wget -q -O /app/iri/bin/iri-1.5.0.jar \
    https://github.com/iotaledger/iri/releases/download/v1.5.0/iri-1.5.0.jar

RUN chown -R root:root /app/iri/

RUN install --directory --owner=root --group=root --mode=0755 /app/iri/conf

COPY iri.ini /app/iri/conf/

USER iri
CMD /usr/bin/java -Xmx4096m -Djava.net.preferIPv4Stack=true -jar /app/iri/bin/iri-1.5.0.jar -c /app/iri/conf/iri.ini --remote
```

**Comment by rajivshah3** [comment]
Could it be related to #844 ?

**Comment by grittibaenz** [comment]
The problem has been somehow resolved with the new `IRI version 1.5.1` and the smaller database. No changes made on my container. I tried using Oracle JDK, OpenJDK, CentOS and/or Ubuntu container images, but it never worked until the new version was release.
